BlueMapCommon/src/main/java/de/bluecolored/bluemap/common/serverinterface/Server.java

```diff
@@ -99,6 +101,9 @@ default Optional<ServerWorld> getServerWorld(Object world) {
     @DebugDump
     Collection<Player> getOnlinePlayers();
 
+    @DebugDump
+    Map<Key, Integer> getSkyBrightness();
```

Comment:
I think this should be something like a `int getSkyBrightness()` or maybe even `int getWorldTime()` in the `ServerWorld` interface instead